recipe/build.sh

```diff
@@ -8,7 +8,7 @@ if [[ "$CONDA_BUILD_CROSS_COMPILATION" == "1" ]]; then
     export BUILD_CC=${CC_FOR_BUILD}
 fi
 
-if [[ "$CONDA_BUILD_CROSS_COMPILATION" == "1"  && "$target_platform" == "osx-arm64" ]]; then
+if [[ "$target_platform" == "osx-64" || "$target_platform" == "osx-arm64"]]; then
```

Comment:
missing a space here.

Reply:
in fact:

` if [[ $target_platform == osx-64 ]] || [[ $target_platform == osx-arm64 ]]; then`